testing/python/raises.py

```diff
@@ -280,7 +280,7 @@ def __class__(self):
 
     def test_raises_context_manager_with_kwargs(self):
         with pytest.raises(TypeError) as excinfo:
-            with pytest.raises(Exception, foo="bar"):  # type: ignore[call-overload]
+            with pytest.raises(Exception, foo="bar"):  # type: ignore[call-overload] # noqa: B017
```

Comment:
Let's change `Exception` to `OSError` or something like that, since the type doesn't matter so we can avoid the noqa.

Reply:
[21283dc](https://github.com/pytest-dev/pytest/pull/11914/commits/21283dc02e98bc1ff3af6f254ec687f853e741fb)